looper/cli_pydantic.py

```diff
@@ -0,0 +1,159 @@
+from dataclasses import dataclass
+import enum
+from typing import Any, Optional
+
+import pydantic
+import pydantic_argparse
+
+from .const import MESSAGE_BY_SUBCOMMAND
+
+
+class Argument(pydantic.fields.FieldInfo):
+    """
+    CLI argument / flag definition
+
+    Naively, one would think one could just subclass `pydantic.Field`,
+    but actually `pydantic.Field` is a function, and not a class.
+    `pydantic.Field()` returns a validated `FieldInfo` instance,
+    so we instead subclass `FieldInfo` directly and validate it in the
+    constructor.
+
+    :param str name: argument name, e.g. "ignore-args"
+    :param str description: argument description, which will appear as the
+        help text for this argument
+    :param Any default: a tuple of the form (type, default_value). If the
+        default value is `...` (Ellipsis), then the argument is required.
+    """
+
+    def __init__(self, name: str, default: Any, description: str) -> None:
```

Comment:
You're right, information like this would go into the `Argument` class, but we'd prefer to add this later, because it's three steps into the future - first, we're focusing on the CLI, then on the HTTP API, and then we can think about functionality to create a web interface. But thanks for pointing us to the corresponding types in `cli_looper.py`!